### ISSUE TITLE
recording app: identification methods can result in an empty serial id now: TEST all identifications methods in all projects

### ISSUE THREAD

**Issue by urapadmin** [arch-kiosk/arch-kiosk-office#2968]
coming out of #2967 and #2956 

Ja. That is the case, unfortunately.
We have to check that all parts of each project that can create a new identifier are still working properly in both normal mode and temporary free input mode.

So, whenever we roll out a recording app version 16.4.4 or higher for the first time for a project all the creation of all these records needs to be checked:

- site
- unit
- feature unit
- survey unit
- legacy recording unit (units with excavation / feature recording)
- creating a feature unit from within a legacy recording unit or a survey unit
- locus
- coordinates
- collected material
- ceramic fragments (SU ceramic analysis)

And again, for normal creation / creation with temporary free input / editing an existing identifier with temporary free input

- [x] RUBAB
- [ ] MADP
- [ ] MKAP
- [x] BUAP
- [ ] ARCH1900
- [ ] KHPP
- [ ] KAMBE
- [ ] PASU
- [ ] USTP
- [ ] URAP
- [ ] ELRI
- [ ] IBASA
- [ ] MHT
- [ ] AAC
- [ ] PACHA
- [x] PVD

**Comment by luizaogs** [comment]
Yikes. Does it make sense to test this in PVD, considering the mess of identification methods? Or is this only for real projects?

**Comment by lbestock** [comment]
This is working in BUAP but BUAP is also rather easy because it's a pure legacy project and so most ID methods are set to free input in any case. But I can make every type of ID listed above as expected, and can change every one of them as expected, too, with the sole exception that I still don't really understand the way the ANC#s work given that they are completely divorced from relationship (in numbering) to any other type of entity. But I think that's nothing to do with this not working.

**Comment by urapadmin** [comment]
> Yikes. Does it make sense to test this in PVD, considering the mess of identification methods? Or is this only for real projects?


ja, why not. If you find a bug in PVD we don't need to encounter it in a real project.

Of those projects that have the exact same identification methods running it is enough to test only one. And we can test this bit by bit. It is a bit before the next projects go into the field again.

**Comment by urapadmin** [comment]
Let's close this and see if any projects report issues
